### ISSUE TITLE
Comments with Quotes Cause Strange HTML Injection

### ISSUE THREAD

**Issue by entozoon** [panoply/vscode-liquid#39]
Input:
```liquid
<html>
  <head>
    {% comment %}
     apostrophe's
    {% endcomment %}
  </head>
  <body>
  </body>
</html>
```
Formatted:
```liquid
<html>
  <head>
    {% comment %}
     apostrophe's
    {% endcomment %}
  </head>
  <body>
  </body>
</html>
  </head>
</html> 
```
And after every format it injects another..
```html
  </head>
</html>
```
..every single time!

It appears to only happen when a comment contains an ' apostrophe or " quote, which is quite bizarre :)

**Comment by panoply** [comment]
Hey @entozoon I am aware of this and next release will fix this issue. It's actually a PrettyDiff issue but with version 2.4 nearing completion a lot of formatting conflicts and issues (like this) will be instead handled by the extensions parser opposed to PrettyDiff. 

Some additional features to come in the next version will be a new and improved way to write comments which takes some minor inspiration from JS Docs.

**Comment by entozoon** [comment]
Sounds amazing, great work @panoply . You rule :)

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)
